frontend/src/app/Routes.tsx

```diff
@@ -21,7 +21,10 @@ const BYONImagesPage = React.lazy(() => import('../pages/BYONImages/BYONImages')
 const NotFound = React.lazy(() => import('../pages/NotFound'));
 
 const Routes: React.FC = () => {
-  const isAdmin = useSelector<State, boolean>((state) => state.appState.isAdmin || false);
+  const [isAdmin, isAllowed] = useSelector<State, [boolean, boolean]>((state) => [
+    state.appState.isAdmin || false,
+    state.appState.isAllowed || false,
+  ]);
```

Comment:
I [improved this logic](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/app/Routes.tsx#L25) with selectors so we didn't have to mess with Redux global state so impurely. Please add `isAllowed` to the [selector](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/redux/selectors/user.ts#L5).

Reply:
Added